code/game/turfs/open/floor/plating/misc_plating.dm

```diff
@@ -196,6 +202,11 @@
 	smoothing_groups = list(SMOOTH_GROUP_TURF_OPEN, SMOOTH_GROUP_FLOOR_ICE)
 	canSmoothWith = list(SMOOTH_GROUP_FLOOR_ICE)
 
+/turf/open/floor/plating/ice/smooth/red
+	icon = 'icons/turf/floors/red_ice.dmi'
+	icon_state = "red_ice-0"
+	base_icon_state = "red_ice"
```

Comment:
What is this? Do we use this anywhere?

Reply:
I was asked to port it, along with some other sprites by Pesto and Kit